zanata-war/src/main/java/org/zanata/webtrans/client/util/JavascriptUtil.java

```diff
@@ -33,6 +35,7 @@
      * @param varName Variable name.
      * @return The value (as a string) assigned to varName.
      */
+    @CoverageIgnore
```

Comment:
That was the first one I put in, before I thought of adding a reason to the annotation.  I'll put it in.